react-ui/src/components/Layouts/InputWorkerTimes/AvailableMeetingHeader.css

```diff
@@ -0,0 +1,23 @@
+.appHeader {
```

Comment:
Hi Kyle,
Make sure to change the file extension to `scss` instead of `css` for consistency.

Reply:
all css files have been changed to scss